packages/orgx/package.json

```diff
@@ -34,6 +34,8 @@
     "hast-util-to-estree": "^2.2.1",
     "periscopic": "^3.0.0",
     "unified": "^10.0.0",
+    "uniorg-parse": "workspace:^",
```

Comment:
@rasendubi I'm not familiar with pnpm, is this meant to be like that when adding packages from the workspace?

Reply:
hm... I'm actually not sure. We'll likely need specific versions when we publish the packages (users of `orgx` through npm won't know what "workspace:^" is)

Reply:
ok, let's merge this and see if publishing works 😅

Reply:
yay, that worked! the published package.json has these versions fixed (instead of "workspace:^")

Reply:
Talking about yolo'ing it :D